specification/commerce/resource-manager/Microsoft.Commerce/preview/2015-06-01-preview/commerce.json

```diff
@@ -88,6 +93,17 @@
               "$ref": "#/definitions/UsageAggregationListResult"
             }
           },
+          "202": {
+            "description": "Response indicating that a request has been accepted for processing. However, the processing has not been completed.",
+            "headers": {
+              "Location": {
```

Comment:
>Location [](http://example.com/codeflow?start=15&length=8)

Do you actually return a location header?  If not, remove this

Reply:
Removed.